tools/interop_matrix/client_matrix.py

```diff
@@ -306,8 +306,12 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
             ("v1.63.3", ReleaseInfo()),
             ("v1.64.1", ReleaseInfo()),
             ("v1.65.0", ReleaseInfo()),
```

Comment:
Please replace the older patch versions. We want to test only the latest patch for every minor version.

Reply:
Done